test/base.test.js

```diff
@@ -382,8 +384,9 @@ test('close with promises', ({ same, plan, pass }) => {
   stream.end()
 })
 
-test('support Transform streams', ({ same, plan, error }) => {
-  plan(7)
+test('support Transform streams', (t) => {
+  // TODO: why isn't `plan: 7` counted correctly?
+  const { deepEqual, ifError } = tspl(t, { plan: 7 })
```

Comment:
I have no idea why this is not getting counted correctly. I can step through all of the assertions and see `actual === plan` is true in `tspl`. @mcollina do you have any clues?